### ISSUE TITLE
Error: other side closed

### ISSUE THREAD

**Issue by iliapnmrv** [softprops/action-gh-release#536]
```yml
Run softprops/action-gh-release@v2
  with:
    tag_name: v2.5.1
    body_path: distribution/whatsnew/whatsnew-ru-RU
    files: android/app/build/outputs/apk/release/apk-v2.5.1-release.apk
 
👩‍🏭 Creating new GitHub release for tag v2.5.1...
⬆️ Uploading apk-v2.5.1-release.apk...
Error: other side closed
```
I have release `v2.5.1-rc.0`, then I want to create stable release `v2.5.1`
My release gets created in GitHub releases (without artifact) but pipeline fails with `Error: other side closed`

Do you have any ideas?

**Comment by paulo-ferraz-oliveira** [comment]
👋 I also use this action in nightly builds and have the following errors happen, aside from the one you mention:

* `Error: write EPIPE` (also mentioned in https://github.com/softprops/action-gh-release/issues/482)
* `Error: write ECONNRESET`

I wonder if all these are somewhat related, if there could be a retry in case of error, or if the issues should somehow be grouped together.

**Comment by AymericFerreira** [comment]
Hello, I also have this error since recently. 
I think it is because I create 2 parallel jobs and they try to access this resource or something else at the same time.
Everytime I retry the failed job it works, and everytime the longer CI is the one failing.

Is there a more elegant way to do this other than serializing my jobs ?

**Comment by paulo-ferraz-oliveira** [comment]
👋 fwiw, I don't run any of my jobs in parallel, and retries also seem to fix the issues consistently.

**Comment by mrutkows** [comment]
Also, seeing this on matrix builds... is there a means to auto-retry (as re-running "failed jobs" manually kind of defeats the purpose of having the automation)?

**Comment by vorporeal** [comment]
This has been happening increasingly frequently for us, and it's at the end of a 40-minute build, so re-runs are quite painful.

Some sort of internal retry logic would be great.  At the very least, would love to know if anyone has figured out a good trick for running the step again if it fails the first time.

**Comment by manuelleduc** [comment]
Hello,
I'm experiencing the same with parallel builds too.

**Comment by chenrui333** [comment]
I could not reproduce or isolate this on current master and the latest 2.5.2 release in the current regression sweep, so I am closing it to keep the backlog focused on confirmed current bugs. If you are still seeing a different current failure, please open a new issue with a minimal repro or sanitized workflow snippet and the failing logs.
